packages/mui-types/src/index.ts

```diff
@@ -84,7 +84,9 @@ export type IfEquals<T, U, Y = unknown, N = never> =
  * This means `typeof value` is not identical to `number | string`
  * @param actual
  */
-export function expectType<Expected, Actual>(actual: IfEquals<Actual, Expected, Actual>): void;
+export declare function expectType<Expected, Actual>(
```

Comment:
Why is `declare` required here?

Reply:
We're only using this for type checking, this is actually never executed. But technically we could make it an empty function as well. This just keeps the behavior exactly the same as it was before, when only exported a declaration file.